app/next-client-app/components/homepage/CTA.tsx

```diff
@@ -0,0 +1,41 @@
+import BoxReveal from "@/components/magicui/box-reveal";
+import { Button } from "@/components/ui/button";
+import { Github, Rocket } from "lucide-react";
+import Link from "next/link";
+
+export default function CallToAction() {
+  const deployLink = process.env.DEPLOY_LINK ?? "";
+  return (
+    <>
+      {/* CTA section */}
+      <div className="flex flex-col items-center text-center px-5 lg:px-0">
+        <BoxReveal boxColor={"#e6a312"} duration={1.0}>
+          <h1 className="sm:text-5xl text-pretty text-3xl">
+            Join our community
+          </h1>
+        </BoxReveal>
+        <BoxReveal boxColor={"#e6a312"} duration={1.0}>
+          <div className="flex mt-3 gap-5 lg:flex-row flex-col">
+            <a
+              href={
+                "https://github.com/Health-Informatics-UoN/Carrot-Mapper/discussions"
+              }
+              target="_blank"
+            >
+              <Button className="lg:text-lg text-md hover:bg-[#333333]/85 dark:hover:bg-gray-200 bg-[#333333] dark:bg-white dark:text-[#333333]">
+                <Github className="mr-2" /> Start a GitHub Discussion
```

Comment:
```suggestion
                <Github className="mr-2" /> GitHub Discussions
``` 
is enough.

But the button icon size should be consistent with the rest of the button icons in the app also. `size-4` I think?